src/rules/core/fields-not-in-model-rule.js

```diff
@@ -251,7 +256,269 @@ module.exports = class FieldsNotInModelRule extends Rule {
     };
   }
 
-  validateField(node, field) {
+  async getContextsAsync(node, field) {
```

Comment:
This is just a copy of `getContextsSync` but with  `await` and `-Async` functions